client/src/contact/contact.yaml

```diff
@@ -0,0 +1,85 @@
+  transform: [markdown]
+  en: |
+    We value your questions and comments. As such, we have provided multiple methods of communication.
+
+
+    If you need to contact us, please email us at questions@tbs-sct.gc.ca.
+  fr: |
+    Vos questions et commentaires nous intéressent. C'est pourquoi nous avons prévu plusieurs méthodes de communication.
+
+
+    Si vous avez besoin de communiquer avec nous, envoyez-nous un courriel à question@tbs-sct.gc.ca.
+
+feedback_title:
+  en: Have Feedback?
+  fr: Avez-vous des commentaires?
+
+feedback_text:
+  transform: [handlebars, markdown]
+  en: |
+    We are always looking for ideas and suggestions to help us improve the current version of the InfoBase. If you would like to share some ideas or comments regarding your experience 
+    with the tool, please take a few minutes to complete our <a href="{{{survey_link}}}" target="_blank" rel="noopener noreferrer">online survey</a>. The feedback you provide will be used
+    to improve the current version of GC InfoBase and prioritize new features.
+  fr: |
+    Nous sommes constamment à la recherche d’idées et de suggestions pour améliorer la version actuelle de l’InfoBase. Le cas échéant, veuillez prendre quelques minutes pour remplir 
+    notre <a href="{{{survey_link}}}" target="_blank" rel="noopener noreferrer">sondage en ligne</a>. En répondant aux quelques questions qui suivent, vous nous aiderez à améliorer
+    l’InfoBase du GC et à fixer l’ordre de priorité pour les nouvelles caractéristiques.
+
+general_enquiries_title:
+  en: General and Technical Enquiries
+  fr: Questions de nature generale et techniques
+
+general_enquiries_text:
+  transform: [handlebars, markdown]
+  en: |
+    Provision of the information requested on this form is voluntary. The information is being collected for the purpose of responding to your query/comments, 
+    and to improve our Internet suite of products and services. Personal information that you provide is protected under the provisions of the Privacy Act. 
+    Your personal information will be stored in {{ext_link "PSU 914 Public Communications" "http://www.infosource.gc.ca/emp/emp03-eng.asp#psu914"}}. 
+    The act provides you with a right of access to your information.
+  fr: |
+    Les réponses à ce formulaire sont données sur une base volontaire. Nous recueillons ces renseignements en vue de répondre à vos demandes et commentaires et 
+    d’améliorer notre série de produits et de services Internet. Vos réponses sont protégées en vertu des dispositions de la Loi sur la protection des renseignements 
+    personnels et seront conservées dans le répertoire des {{ext_link "POU 914 Communications publiques" "http://www.infosource.gc.ca/emp/emp03-fra.asp#pou914"}}. 
+    La Loi vous donne le droit de consulter vos renseignements personnels.
+
+general_enquiries_contact:
+  transform: [markdown]
+  en: |
+    Email: <a href="mailto:publicenquiries-demandesderenseignement@tbs-sct.gc.ca">Public Enquiries</a><br/>
```

Comment:
If I remember right, two space at the end of a line is markdown syntax for a `br`. Give that a try (although honestly I don't love that syntax, since it's invisible; maybe explicit `br` tags is better)